storage/csi-snapshot.md

```diff
@@ -0,0 +1,120 @@
+---
+title: CSI Snapshots
+authors:
+  - "@chuffman"
+reviewers:
+  - "@gnufied”
+  - “@jsafrane”
+approvers:
+  - "@..."
+creation-date: 2019-09-05
+last-updated: 2019-09-25
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# csi-snapshots
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs] 
+
+## Summary
+We want to include the upstream CSI Snapshot feature in OpenShift.
+
+## Motivation
+
+* This feature has been requested by both the OCS and CNV teams.
+* Snapshotting is an incredibly useful function when it comes to system changes in development areas, as it allows rapid rollback to preview dev versions
+* Container development benefits from filesystems that provide snapshot functionality
+
+### Goals
+
+* Backport and include the external-snapshotter sidecar.
```

Comment:
I think we don't need to backport anything, we just pick upstream 1.16 snapshotter (and provisioner) and release it downstream.

Reply:
Talked to @bertinatto, what we want is "rebase downstream csi-external-snapshotter and csi-external-provisioner to the upstream one released for Kubernetes 1.16"

Reply:
This sentence has been updated to indicate that we're rebasing the upstream content.